config/katello.yaml.example

```diff
@@ -1,4 +1,12 @@
 :katello:
+  :content_types:
+    :yum: true
+    :file: true
+    :deb: true
+    :puppet: true
+    :docker: true
+    :ostree: true
```

Comment:
I think you should put some comments. Namely that the default is that all registered content types are allowed. Optionally, that the Katello repository types are located and registered from `/lib/katello/repository_types/*.rb`.

Reply:
I mostly wanted a comment saying that the default is enabled: true if content_types array is empty/nonexistant